packages/components/src/FilterBar/FilterBar.test.js

```diff
@@ -131,14 +131,17 @@ describe('FilterBar', () => {
 	it('should call onToggle on ENTER keydown', () => {
 		// given
 		const props = { ...defaultProps };
-		const filterInstance = mount(<FilterBarComponent {...props} id={'filter'} />);
+		const filterInstance = mount(<FilterBarComponent {...props} id="filter" />, {
+			attachTo: document.body,
```

Comment:
We select the activeElement. With talend/scripts version of jest, we need to attach the component to the jsdom.